src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx

```diff
@@ -349,7 +350,12 @@ export default function DownloadDialog({
           return
         }
         setDlcsToInstall(
-          (gameInstallInfo?.game.owned_dlc || []).map((dlc) => dlc.app_name)
+          (gameInstallInfo &&
+            'game' in gameInstallInfo &&
+            (gameInstallInfo?.game.owned_dlc || []).map(
+              (dlc) => dlc.app_name
+            )) ||
+            []
```

Comment:
This is a little hard to read, perhaps factor this out into a separate variable? E.g.
```ts
const dlcs = []
if (gameInstallInfo && ...) {
  gameInstallInfo.game.owned_dlc.forEach((dlc) => dlcs.push(dlc.app_name))
}
setDlcsToInstall(dlcs)